hil-test/.cargo/config.toml

```diff
@@ -1,6 +1,9 @@
 [target.'cfg(target_arch = "riscv32")']
 runner    = "probe-rs run --preverify"
 rustflags = [
+    "--cfg", "embedded_test",
+    "--check-cfg", "cfg(embedded_test)",
```

Comment:
This is a nice idea, but shouldn't we place the `--check-cfg` into `esp-bootloader-esp-idf`'s build script?

Reply:
You were faster 😅